projects/igniteui-angular/src/lib/grid/grid.component.ts

```diff
@@ -1189,32 +2279,68 @@ export class IgxGridComponent implements OnInit, OnDestroy, AfterContentInit, Af
         this._moveColumns(column, dropTarget);
     }
 
+    /**
+     * Goes to the next page of the `IgxGridComponent` if it's not the last.
```

Comment:
if possible
if the grid is not already at the last page
something like this because your wording is confusing - literally it means it will move to the next page if the next page is not the last page